src/app/static/src/js/components/reportLog/runningReportsList.vue

```diff
@@ -62,6 +62,11 @@
                 return this.reports.sort((a, b) => a.date.localeCompare(b.date)).reverse();
             }
         },
+        mounted(){
+            if (this.initialSelectedKey){
+                this.query = this.reports.filter(report => report.key === this.initialSelectedKey)[0]["name"]
```

Comment:
`find()` is more idiomatic (and efficient: it returns first match without filtering the whole array):
```suggestion
                this.query = this.reports.find(report => report.key === this.initialSelectedKey).name
```